<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addnewroom</class>
 <widget class="QDialog" name="addnewroom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add new room</string>
  </property>
  <widget class="QPushButton" name="Add">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBut">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="ID_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Room description:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="room_des">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max people:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_people">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>42</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cost rate (price per night):</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_cost">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>2147483647</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
